doc/uploads/plantuml/ara_sequence_diagram.puml

```diff
@@ -0,0 +1,18 @@
+@startuml
+group upload zip (ExecutionRessource)
+  ARA <- CI : sending ZIP
+  note left: calling the resource /upload
+  group analyzing report (ExecutionService)
+    ARA -> ARA : get report details (executions) from zip
+    ARA -> ARA : save details in the server ARA is deployed on
+    note right: saved in the ara-server/path/{{project}}/{{branch}}/{{cycle}}
```

Comment:
Useless comment since you've explained it already in the `process.adoc` file

Reply:
done